### ISSUE TITLE
Please release a new version due to a high vulnerability

### ISSUE THREAD

**Issue by sermayoral** [hassio-addons/addon-nginx-proxy-manager#470]
The NPM version included in the last release has a vulnerability: 
https://www.cvedetails.com/cve/CVE-2023-27224/

The NPM version included in the master branch is OK 😊.

Thanks!!

**Comment by 1aranzant** [comment]
I don't think this is maintained anymore...

**Comment by frenck** [comment]
> I don't think this is maintained anymore...

Really? Did you even look?

![CleanShot 2023-10-18 at 23 15 35@2x](https://github.com/hassio-addons/addon-nginx-proxy-manager/assets/195327/d1c6e926-2f39-41a6-aa8b-a720fae9864c)

**Comment by 1aranzant** [comment]
@frenck my bad, I was relying on the HA page which clearly states that it isn't maintained since 2022... sorry !
![image](https://github.com/hassio-addons/addon-nginx-proxy-manager/assets/17981079/945df3af-314d-427f-8757-ff70fc3cf834)

**Comment by Glennmen** [comment]
@frenck Latest release version is very old and stil requires MariaDB, when will there be a new version?
<img width="494" alt="CleanShot 2023-10-25 at 22 23 27@2x" src="https://github.com/hassio-addons/addon-nginx-proxy-manager/assets/11406433/a6de9333-b5a1-4f9f-9502-c80c591e21a0">

**Comment by BjoernPetersen** [comment]
Still relevant, since the latest release is still v0.12.3

**Comment by Pikachews** [comment]
@frenck Sorry to bother you, but any chance a new release will be made? Would use some of the new changes (removing MariaDB) and vuln fixes. 🙏

**Comment by 1aranzant** [comment]
he's too busy condescendingly telling others to look at the add-on page which clearly states the add-on is not maintained since 2022...

**Comment by Glennmen** [comment]
It was only today that I learned how to install "edge" versions of addons. For anybody willing to get the latest version without waiting for a new stable release version.
However risking that it might be unstable and give issues.

You can add this repository:
`https://github.com/hassio-addons/repository-edge`

**Comment by frenck** [comment]
Don't use the edge version. It will be broken.

**Comment by Glennmen** [comment]
@frenck Thanks the comment ❤️ 
I was planning on migrating from the default nginx addon to nginx-proxy-manager addon this weekend but will hold on till the new version is released 👍

**Comment by frenck** [comment]
This should be the final step: #505

Needs a lot of testing, but... should get this add-on in a much better place. No weird workarounds, no external databases. Add-on backups include everything. Just run, use.

**Comment by frenck** [comment]
Small update: Everything has been solid in testing. Running multiple instances on multiple architectures on different locations; all behave as expected.

I found a small bug with the handling of plugins for certbot when using a DNS challenge on some provides that require on-the-fly installation of plugins. I've corrected this and am currently building a new dev/test version to ensure these tests pass as well.

If all goes well, I might start shipping a new version of the add-on today or tomorrow.
